shared/constants/network.ts

```diff
@@ -359,7 +359,8 @@ export const AETH_TOKEN_IMAGE_URL = './images/arbitrum.svg';
 export const FTM_TOKEN_IMAGE_URL = './images/fantom-opera.svg';
 export const HARMONY_ONE_TOKEN_IMAGE_URL = './images/harmony-one.svg';
 export const OPTIMISM_TOKEN_IMAGE_URL = './images/optimism.svg';
-export const PALM_TOKEN_IMAGE_URL = './images/palm.svg';
+export const PALM_TOKEN_IMAGE_URL_DARK = './images/palm-dark.svg';
+export const PALM_TOKEN_IMAGE_URL_WHITE = './images/palm-white.svg';
```

Comment:
Were the icons sourced from here? https://stormy-dracorex-499.notion.site/Palm-Ecosystem-Brand-Logos-96ac0235320c4b629e8b01d49cb45798

Is the light + dark theme support because the icons are transparent?

Another approach could set the background color to black in the SVG file, so that it works in both themes.

Reply:
hey @bergeron ,
i took the icons from `Primary Logo Black` and `Primary Logo White` section ont the table.

Reply:
I'm wondering if the theme feature is actually needed.  Is there a reason we shouldn't just use one of the primary logos?  `style="background-color:..."` could be added if it's the transparent backgrounds we're fighting.

Reply:
eeem i think i got what you mean, QQ we still need to get the used theme to change the background color on the solution you just described ? another thing, will this be applicable only for palm logo ?

Reply:
@bergeron i just added a more concise version based on the background color like you suggested

Reply:
My comment may not have been clear.  I was thinking of adding `background-color: black/white` on the `<svg>` element in the icon file.  That would avoid the transparent background and make it readable on both themes.

Reply:
@bergeron done